cmd/zoekt-git-index/main.go

```diff
@@ -23,13 +23,13 @@ import (
 	"strings"
 
 	"github.com/dustin/go-humanize"
+	"github.com/sourcegraph/zoekt/internal/ctags"
 	"go.uber.org/automaxprocs/maxprocs"
 
 	"github.com/sourcegraph/zoekt/internal/profiler"
 
 	"github.com/sourcegraph/zoekt/cmd"
-	"github.com/sourcegraph/zoekt/ctags"
-	"github.com/sourcegraph/zoekt/gitindex"
+	"github.com/sourcegraph/zoekt/internal/gitindex"
```

Comment:
nit: it might be good to configure `goimports` or something similar to keep imports tidy :)

Reply:
Thanks, I hadn't checked "Optimize imports on save" for the zoekt repo.